tests/test_fringes.py

```diff
@@ -252,6 +260,15 @@ def testRunDataRef(self, pedestal=0.0, stddevMax=1.0e-4):
         mi -= afwMath.makeStatistics(mi, afwMath.MEAN).getValue()
         self.assertLess(afwMath.makeStatistics(mi, afwMath.STDEV).getValue(), stddevMax)
 
+    def test_readFringes(self):
+        r"""Test that fringes can be successfully accessed from the butler.
+        """
+        task = FringeTask()
+        dataRef = isrMock.DataRefMock()
+
+        result = task.readFringes(dataRef, assembler=None)
+        assert result
```

Comment:
What are you checking? You have a `dataRefMock`, so I assume you can test that it returned the expected thing, instead of just not being None.